### PR TITLE
winlibs-mingw-ucrt: Add version 11.2.0-13.0.0-9.0.0-ucrt-r2

### PR DESCRIPTION
**Pull request by Somiona**
I'm not sure about how scoop repo works, but I tried and this file do work for me.

### CODE REVIEW

**Comment by chawyehsu** [comment]
Great idea, would you please make `winlibs-mingw-ucrt.json` be the one w/o llvm, and create a new manifest with name `winlibs-mingw-llvm-ucrt.json` for the one w/ llvm?

**Comment by Somiona** [comment]
> Great idea, would you please make `winlibs-mingw-ucrt.json` be the one w/o llvm, and create a new manifest with name `winlibs-mingw-llvm-ucrt.json` for the one w/ llvm?

Sure, new changes is avaliable for review. I do also modified both English and Chinese readme file to include those new versions of *winlibs*.

**Review by chawyehsu — APPROVED** [review]

File: bucket/winlibs-mingw-ucrt.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {
     "homepage": "https://winlibs.com/",
-    "description": "winlibs standalone build of GCC compiler and MinGW-w64",
+    "description": "winlibs standalone build of GCC compiler and MinGW-w64 with llvm",
```
Comment (rasa) [inline bucket/winlibs-mingw-ucrt.json:3]: This manifest does not include LLVM, right?

Comment (chawyehsu) [reply bucket/winlibs-mingw-ucrt.json:3]: Good catch, missed that description check...

Comment (Somiona) [reply bucket/winlibs-mingw-ucrt.json:3]: Oh yes I forgot to change this line. I will create a new PR fixing this shortly

Comment (Somiona) [reply bucket/winlibs-mingw-ucrt.json:3]: https://github.com/chawyehsu/dorado/pull/446

**Pull request merged**